examples/inertia_relief/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+blt_add_executable( NAME inertia_relief_example
+	            SOURCES inertia_relief_example.cpp
+		    OUTPUT_DIR ${EXAMPLE_OUTPUT_DIRECTORY}
+		    DEPENDS_ON serac_physics serac_mesh_utils serac_functional continuation_solver)
+
+install(
```

Comment:
Can you add a test for this as well? There are examples in the other example directories

Reply:
@white238 -- I see serac_add_example_test in the examples/*/CMakeLists.txt files. For my own understanding are these tests just making sure that compiled examples run without error?

Reply:
Correct. Unless you want to add some error checking into it.